src/main/java/org/opensearch/security/privileges/PrivilegesEvaluator.java

```diff
@@ -104,7 +104,7 @@ public class PrivilegesEvaluator {
     private static final WildcardMatcher ACTION_MATCHER = WildcardMatcher.from("indices:data/read/*search*");
 
     private static final Pattern DNFOF_PATTERNS = Pattern.compile(
-        "indices:(data/read/.*|(admin/(mappings/fields/get.*|shards/search_shards|resolve/index)))"
+        "indices:(data/read/.*|(admin/(mappings/fields/get.*|shards/search_shards|resolve/index))|(monitor/((settings/get)|(stats))))"
     );
```

Comment:
Also not sure if this covers all applicable CAT API calls - is there anymore calls we want to enable DNFOF for in this same PR?

Reply:
Can you add a unit test around the what the pattern matches / doesn't match?

Reply:
Added a unit test in: [92d475b](https://github.com/opensearch-project/security/pull/3236/commits/92d475bf02c966ae6359eeb222f234cd6766a003), only question is if it is ok living in the existing DNFOF file or not.

Reply:
But my original question still stands - do we think that a list would be cleaner/clearer than matching against a regex pattern?

Reply:
All test under `src/integrationTest` create a fresh cluster with each test (or if not they should move...)  When I say unit test it could be a test that just accesses the regex static, not even making an instance of PrivilegesEvaluator

Reply:
Moved in [4185872](https://github.com/opensearch-project/security/pull/3236/commits/41858729da8b2af78dd7bde4267613deaf611fd2)